tox.ini

```diff
@@ -28,12 +29,16 @@ setenv =
 deps =
     -r requirements/tests.txt
     dj22: Django<2.3
+    dj30: Django<3.1
+    dj31: Django<3.2
     dj32: Django<3.3
+    dj40: Django<4.1
+    dj41: Django<4.2
```

Comment:
Only Django 3.2 LTS, 4.0 and 4.1 are supported, so let's drop the others.

* https://www.djangoproject.com/download/#supported-versions
* https://endoflife.date/django

Reply:
Modified as requested